tests/common/plugins/ptfadapter/__init__.py

```diff
@@ -77,14 +78,25 @@ def get_ifaces_map(ifaces, ptf_port_mapping_mode):
     """Get interface map."""
     sub_ifaces = []
     iface_map = {}
+    used_index = set()
+    backplane_exist = False
     for iface in ifaces:
         iface_suffix = iface.lstrip(ETH_PFX)
         if "." in iface_suffix:
             iface_index = int(iface_suffix.split(".")[0])
             sub_ifaces.append((iface_index, iface))
+        elif iface == BACKPLANE:
+            backplane_exist = True
         else:
             iface_index = int(iface_suffix)
             iface_map[iface_index] = iface
+            used_index.add(iface_index)
+
+    count = 1
+    while count in used_index:
+        count = count + 1
+    if backplane_exist:
+        iface_map[count] = "backplane"
```

Comment:
Hi @eddieruan-alibaba, could you please help explain why `backplane` is added to `iface_map`?